distributed/utils.py

```diff
@@ -386,7 +386,7 @@ def key_split_group(x):
     typ = type(x)
     if typ is tuple:
         return x[0]
-    elif typ is str:
+    elif typ is unicode or typ is str:
```

Comment:
In Python 2, wouldn't this case have handled `bytes` and the case below would only handle `bytes` on Python 3? Was that unintentional? I was trying to replicate the behaviour of the original.